cmd/kusk/cmd/install.go

```diff
@@ -162,6 +236,65 @@ var installCmd = &cobra.Command{
 
 		if !noDashboard {
 			kuskui.PrintStart("installing Dashboard...")
+			manifest, err := os.ReadFile(filepath.Join(dir, manifests_dir, "dashboard_staticroute.yaml"))
```

Comment:
I don't understand why you are applying this in the `!noDashboard` block.

Shouldn't it be the reverse?